symplyphysics/laws/waves/wavelength_from_velocity_and_frequency.py

```diff
@@ -0,0 +1,37 @@
+from sympy import pi
+from symplyphysics import (
+    symbols, Eq, pretty, solve, Quantity, units,
+    validate_input, validate_output, expr_to_quantity
+)
+
+# Description
+## The main characteristics of any wave are it's frequency, wavelength, magnitude and propagation speed.
+## Magnitude is power characteristic of wave.
+## Other characteristics are dependent from each over.
+
+## Definition: λ = V / (n * f)
```

Comment:
Clearly this law is very similar to the previous one. Decide which one is more general and derive more specific law from it. Or add `TODO` if there are some missing parts.